### DISCUSSION TITLE
certificates for CNAME records

### DISCUSSION THREAD

**Discussion by MikalaiHryniuk** [polymind-inc/acmebot#523] [Ideas]
Is it possible to implement a solution to generate certificates not only for A-records but also for CNAMES?

**Comment by shibayan** [comment]
It is already supported regardless of A-record or CNAME.

**Reply by MikalaiHryniuk** [reply]
Can I create a certificate for a domain that only has a CNAME record for my A-record in Azure DNS? the CNAME domain is in cloudflare

**Reply by MikalaiHryniuk** [reply]
And I don't have access to the cloudflare

**Reply by shibayan** [reply]
That's the CNAME delegation in #159, which has nothing to do with A records and is completely different.

**Reply by MikalaiHryniuk** [reply]
Thank you. I'm sorry, I worded my question a little wrong
